docs/tutorials/release-using-channels.md

```diff
@@ -0,0 +1,63 @@
+> !!! This documentation is in "beta" and needed to be tested !!!
+
+# Release using channels / Auto-updates with channels
```

Comment:
Please in the future use form  "Header Must Use Capital Case".

Reply:
Thanks for the comment, I've change this.